docs/source/get_started/computers.rst

```diff
@@ -203,79 +200,69 @@ The configuration of computers happens in two steps.
         aprun -n {tot_num_mpiprocs}
         poe
 
-   * **Text to prepend to each command execution**: This is a multiline string,
-     whose content will be prepended inside the submission script before the
-     real execution of the job. It is your responsibility to write proper ``bash`` code!
-     This is intended for computer-dependent code, like for instance loading a
-     module that should always be loaded on that specific computer. *Remember*
-     *to end the input by pressing* ``<CTRL>+D``.
-     A practical example::
+  * **Default number of CPUs per machine**: The number of MPI processes per machine that
+    should be executed if it is not otherwise specified. No default value is specified by 0. 
+   
+  At the end, you will land on an vim editor page, with a summary of the configuration up to this point, 
+  and the possibility to add content that will be executed either
+  *before* the actual execution of the job (under the 'pre-execution script' lines) or *after* the 
+  script submission (under the 'Post execution script' lines). 
+  These additional lines need to be proper ``bash`` code, that is intended for computer-dependent code
+  (for instance a module that should always be loaded). For example::
 
         export NEWVAR=1
         source some/file
 
-     A not-to-do example::
-
-       #PBS -l nodes=4:ppn=12
-
-     (it's the plugin that will do this!)
-
-   * **Text to append to each command execution**: This is a multiline string,
-     whose content will be appended inside the submission script after the
-     real execution of the job. It is your responsibility to write proper ``bash`` code!
-     This is intended for computer-dependent code. *Remember*
-     *to end the input by pressing* ``<CTRL>+D``.
-   
-  At the end, you will get a confirmation command, and also the ID in the
-  database (``pk``, i.e. the principal key, and ``uuid``).
+  .. note:: Pre-execution commands should **not** be used to define resources that are handled by the scheduler plugin, 
```

Comment:
```suggestion
  .. note:: Don't specify settings here that are specific to a code or calculation -- you can set further pre-execution commands at the ``Code`` and ``CalcJob`` level.
```

Reply:
I think that Daniele meant that one should not add e.g. `$SBATCH -n 2` instructions. But you are also right.

Reply:
Yes, @borellim, exactly. I guess both can be left there @ltalirz ?